packages/webpack-plugin/src/index.js

```diff
@@ -29,14 +29,13 @@ class LoadablePlugin {
     const result = JSON.stringify(stats, null, 2)
 
     if (this.opts.outputAsset) {
-      hookCompiler.assets[this.opts.filename] = {
-        source() {
-          return result
-        },
-        size() {
-          return result.length
-        },
-      }
+      const outputPath = nodePath.resolve(
+        this.compiler.options.output.path,
+        this.opts.filename,
+      )
+      const file = fs.createWriteStream(outputPath)
+      file.write(result)
```

Comment:
this is definitely not a correct way to handle this moment. You should not `write` file in order to handle deprecation warning, especially if that functionality already exists via writeAssetsFile.

Ask yourself why there were two ways of doing this?

Let's proceed with, which addresses the same problem without removing breaking the contract between loadable and webpack. 
https://github.com/gregberge/loadable-components/pull/676/files#diff-ca9a5ec2c1a865ca7d717cfaed22c11c91459fefce687dccc92ca910c1409c5eL32

Reply:
https://github.com/gregberge/loadable-components/pull/676/files#diff-ca9a5ec2c1a865ca7d717cfaed22c11c91459fefce687dccc92ca910c1409c5eL32 This version works fine for me now, it bundle files without warning.